helmfile.d/helmfile-03.init.yaml

```diff
@@ -31,9 +31,20 @@ releases:
     labels:
       pkg: loki
     <<: *raw
-  - name: gatekeeper-operator
-    installed: {{ $c | get "gatekeeper-operator.enabled" false }}
+  - name: gatekeeper
```

Comment:
In the helmfiles, can you please refer to the chart name? So in this case we will still use the "gatekeeper-operator" chart (and the release name should reflect that always).

So all that changes is the prop in the values.

Reply:
The same as istio-operator

Reply:
ok